atomic_reactor/plugins/exit_koji_promote.py

```diff
@@ -451,10 +451,8 @@ def add_log_type(output):
 
         # Read config from the registry using v2 schema 2 digest
         registries = self.workflow.push_conf.docker_registries
-        if registries:
-            config = copy.deepcopy(registries[0].config)
-        else:
-            config = {}
+        assert registries, "registry config is required"
```

Comment:
How likely is this to happen? Shouldn't be this rather if statement with exception? (I'm very skeptical to asserts out of tests)

Reply:
Removed it.